src/commands/stac-setup/stac.setup.ts

```diff
@@ -102,7 +102,7 @@ export const commandStacSetup = command({
         region: args.region,
         geographicDescription: args.geographicDescription,
         date: date,
-        gsd: args.gsd,
+        gsd: formatGsd(args.gsd),
```

Comment:
Do we want to add more verification/check on the inputs? e.g. if `gsd` is passed with any random string, what happens to the slug?